platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -208,10 +224,11 @@ class Summary(
                     appendLine("plus ${reportableProblemCauseCount - MAX_CONSOLE_PROBLEMS} more problems. Please see the report for details.")
                 }
             }
+            val hasIncompatibleTasks = incompatibleTasksCount > 0
             htmlReportFile?.let {
                 appendLine()
-                if (!hasReportableProblems) {
-                    // Some tests parse this line.
+                if (hasIncompatibleTasks && !hasReportableProblems) {
+                    // Some tests parse this line, you may need to change them if you change the message.
```

Comment:
💭  This comment would be helpful if this string were a rare occasion when we parse output in tests. Unfortunately, we parse basically every CC related string. Having a comment next to only one of them is unnecessary. Besides, if the tests are not adjusted, they will fail.